src/site/content/en/blog/file-handling/index.md

```diff
@@ -128,7 +128,7 @@ This is for a hypothetical application that handles comma-separated value (`.csv
 scalable vector graphics (`.svg`) files at `/open-svg`, and a made-up Grafr file format with any of
 `.grafr`, `.graf`, or `.graph` as the extension at `/open-graf`.
 
-{% Aside %} For this declaration to have any effect, the application must be installed. You can
+{% Aside 'warning' %} For this declaration to have any effect, the application must be installed. You can
```

Comment:
```suggestion
{% Aside %} For this declaration to have any effect, the application must be installed. You can
```

Reply:
The [handbook](https://web.dev/handbook/web-dev-components/#asides) says that a warning should, "tell the reader not to do something." I actually think this heuristic is weak. Since long before Google, I've reserved warnings for things that can cause severe problems like data loss or application crashes.